subsys/mgmt/mcumgr/smp_udp.c

```diff
@@ -121,6 +121,9 @@ static void smp_udp_receive_thread(void *p1, void *p2, void *p3)
 
 			/* store sender address in user data for reply */
 			nb = mcumgr_buf_alloc();
+			if (!nb) {
+				continue;
+			}
```

Comment:
I am not sure whether it is ok, because this will be skipping some data, as continue will execute `recvfrom` , and then some buffer will be allocated and the `zephyr_smp_rx_req` will be called with some missing bytes in front, shouldn't here be at least some log?

Reply:
Quite frankly i dont have much time i can spent on that and think about the correct solution so i thought its better to put anything than nothing here. I could add `LOG_ERR`, would that be okay?

Reply:
Yes, but also add some note and if it would be possible, report issue in Zephyr so that we would not miss it an have a chance to discuss possible problems.